src/participants/create_project.py

```diff
@@ -104,7 +104,11 @@ def get_repolinks(self, wid, project):
                 repolinks[repo].append(arch)
             if not repolinks[repo]:
                 del repolinks[repo]
-        return repolinks
+        if repolinks:
+            return repolinks
+        else:
+            # Creating a project with no repos makes no sense
+            raise RuntimeError("No suitable repos found in %s (must contain an arch in the name)" % project)
```

Comment:
I don't agree with raising here.

1) get_repolinks provides descriptive data
2) repolinks could be empty because a repo or arch is excluded
3) Per se, there is nothing wrong with creating a project without any repository

I suspect you have a more specific failure case in mind you want to guard against?